src/Sylius/Bundle/AdminApiBundle/Tests/DependencyInjection/SyliusAdminApiExtensionTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminApiBundle\Tests\DependencyInjection;
+
+use Doctrine\Bundle\MigrationsBundle\DependencyInjection\DoctrineMigrationsExtension;
+use FOS\OAuthServerBundle\DependencyInjection\FOSOAuthServerExtension;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Sylius\Bundle\AdminApiBundle\DependencyInjection\SyliusAdminApiExtension;
+use SyliusLabs\DoctrineMigrationsExtraBundle\DependencyInjection\SyliusLabsDoctrineMigrationsExtraExtension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+
+final class SyliusAdminApiExtensionTest extends AbstractExtensionTestCase
+{
+    /** @var array */
+    private $config =
+        ['sylius_admin_api' =>
+            ['resources' =>
+                ['api_user' =>
+                    ['classes' =>
+                        ['model' => 'Sylius\Component\Core\Model\AdminUser'],
+                    ],
+                ],
+            ],
+        ]
+    ;
+
+    /**
+     * @test
+     */
+    public function it_autoconfigures_prepending_doctrine_migration_with_proper_migrations_paths(): void
+    {
+        $this->configureContainer();
+
+        $this->customLoad($this->config);
```

Comment:
Why do we even pass this config as a function? Will it change between calls? Won't it be better to just hardcode in `customLoad` function?